packages/components/Avatar/src/useAvatar.ts

```diff
@@ -161,6 +163,29 @@ export const removeTitlesFromName = (words: string[]): string[] => {
   return words.filter((word) => !titles.has(word));
 };
 
+/**
+ * A function which verifies whether words array contains multi-word title
+ *
+ * @param name
+ * @returns name without multi-word titles
+ */
+const removeMultiWordTitles = (name: string, idx: number = 0): string => {
+  if (!isMultiWordName || idx >= multiWordTitles.length) return name;
+  const title = multiWordTitles[idx];
+  const _name = name.indexOf(title) !== -1 ? name.replace(title, '') : name;
+  return removeMultiWordTitles(_name, idx + 1);
```

Comment:
I think we can bail from this as soon as we find a match. It probably won't be very common to see more than one multi-title title in someone's name, so doing this would save on perf.

Reply:
Yeah, I was thinking what would be better. Because one multi-word title is rare but double multi-word title can be only QA edge case :)